assets/javascripts/kitten/components/atoms/avatar-with-text-and-badge/index.js

```diff
@@ -65,7 +65,7 @@ const StyledWrapper = styled.div`
     position: absolute;
     top: ${pxToRem(-4)};
     right: ${pxToRem(-8)};
-    border: ${pxToRem(2)} solid white;
+    border: ${pxToRem(1)} solid white;
```

Comment:
Il me semblait qu'on allait faire un token spécifique pour la largeur du border, façon :

```suggestion
    border: var(--border-width) solid white;
```

Reply:
Allez !